### PR TITLE
Edit Profile Style Changes

### PR DESCRIPTION
**Pull request by Seroxdesign**
## Overview

**What features/fixes does this PR include?**

This PR implements the design for a single page profile edit layout.

**Please provide the GitHub issue number**

Closes #

#1233 

## Follow up Improvement Ideas

- [ ] please list any improvement/ideas

## Implementation

**Describe technical (nontrivial / non-obvious) parts of your code**

**Side effects**

## Assets

[Include screenshots/videos if it makes reviewing easier.]

### CODE REVIEW

**Comment by Seroxdesign** [comment]
@alalonde ready for review

**Comment by alalonde** [comment]
@davort Can you look at the deployed instance corresponding to this PR? Any feedback beyond what I mentioned above?

**Comment by dysbulic** [comment]
@Seroxdesign & @nitegeist, at @davort's request we're going to remove the `emoji`, `location`, & `pronouns` fields.

The `username` field is to be switched to "Profile URL" and the field will begin with the uneditable text  `https://metagame.wtf/players/` followed by a blank for the username to show what the username is for.

@alalonde created a separate issue for this at #1559.

**Comment by nitegeist** [comment]
@dysbulic understood. I'll make those changes and push soon

**Comment by davort** [comment]
> @davort Can you look at the deployed instance corresponding to this PR? Any feedback beyond what I mentioned above?

Sorry for the super delayed response.
Certainly no showstoppers, but I'd suggest offering a more descriptive and, arguably more useful placeholder text for the first two fields.

A screenshot of the deployed instance:
CleanShot 2023-05-26 at 10 13 04

I'd suggest replacing:
* `Imma User` in the first field with `e.g., Meta Player 10x!`
* `i-am-a-user` in the second field with `e.g., meta_player-10x`

**Comment by nitegeist** [comment]
I've removed the `location`, `emoji` & `pronouns` fields and I've also added the more descriptive placeholders.

**Comment by alalonde** [comment]
@nitegeist @Seroxdesign Pleaes address my comments from a few weeks ago about commented out code

**Comment by alalonde** [comment]
> @alalonde if you're asking me to remove the commented code then I'll do that. Not sure where I can find your comment from weeks ago at this point.

Sorry, I forgot to finalize my review. Please address the remaining comments and then rebase

File: packages/web/components/Player/Profile/EditAvatarImage.tsx [hunk 1]
```diff
@@ -113,26 +110,78 @@ export const EditAvatarImage = forwardRef<
...
-              position="absolute"
-              top={0}
-              bottom={0}
-              left={0}
-              right={0}
-              opacity={0}
-              onFocus={() => setActive(true)}
-              onBlur={() => setActive(false)}
-            />
+            // <Input
```
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/Player/Profile/EditAvatarImage.tsx:113]: Deleted commented code.

File: packages/web/components/EditProfileModal.tsx [hunk 2]
```diff
@@ -398,36 +386,15 @@ export const EditProfileModal: React.FC<EditProfileModalProps> = ({
...
-                  </Box>
-                </FormControl>
+              <GridItem flex={1}>
+                <EditDescription />
               </GridItem>
               <GridItem flex={1} alignItems="center">
                 <FormControl isInvalid={!!errors.availableHours}>
                   <Label htmlFor="availableHours">Availability</Label>
+                  <FormHelperText pb={3} color="white">
+                    Maximum amount of time you can commit to MetaGame.
```
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/EditProfileModal.tsx:407]: Replaced with copy from `SetupAvailability`

File: packages/web/components/EditProfileModal.tsx [hunk 3]
```diff
@@ -469,6 +436,36 @@ export const EditProfileModal: React.FC<EditProfileModalProps> = ({
                   </Box>
                 </FormControl>
               </GridItem>
+              <GridItem flex={1} alignItems="center">
+                <FormControl isInvalid={!!errors.timeZone}>
+                  <Label htmlFor="name">Time Zone</Label>
+                  <FormHelperText pb={3} color="white">
+                    Maximum amount of time you can commit to MetaGame.
```
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/EditProfileModal.tsx:454]: Removed.

File: packages/web/components/Player/Profile/EditBackgroundImage.tsx.orig
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/Player/Profile/EditBackgroundImage.tsx.orig:1]: Deleted.

File: packages/web/components/Player/Profile/EditAvatarImage.tsx.orig
[the remark this answers is not in the document]
Comment (nitegeist) [reply packages/web/components/Player/Profile/EditAvatarImage.tsx.orig:1]: Deleted.

**Pull request merged**